kafka-connect-s3/src/test/java/io/confluent/connect/s3/TopicPartitionWriterTest.java

```diff
@@ -72,6 +81,7 @@
 public class TopicPartitionWriterTest extends TestWithMockedS3 {
   // The default
   private static final String ZERO_PAD_FMT = "%010d";
+  private enum KVHType { KEYS, HEADERS, VALUES }
```

Comment:
I thought we had a checkstyle rule for naming that used consecutive upper case letters. I guess it's not run in tests. 

```suggestion
  private enum RecordElement { 
  KEYS, 
  HEADERS, 
  VALUES 
  }
```
(please fix alignment, github editing is not great for that).